dockerfiles/README.md

```diff
@@ -0,0 +1,35 @@
+####Docker 
+Docker is an open-source project that automates the deployment of applications inside software
+containers, by providing an additional layer of abstraction and automation of operating-system-level
+virtualization on Linux.
+<https://www.docker.com/what-docker>
+
+####Dockerfiles for Fedora and Ubuntu
+The two docker files, Dockerfile-fedora and Dockerfile-ubuntu provide quick start
+docker containers with an running instance of qpid dispatch router.
+
+The dockerfiles follow these steps before launching the dispatch router
+
+* Downloads all the fedora or ubuntu specific dependencies needed for building apache qpid-proton.
+* Downloads the source code of qpid-proton to /main/qpid-proton
+* Builds and installs qpid-proton using gcc. /usr/local/lib64 is the folder in which the qpid-proton artifacts are installed.
+* Downloads the source code of qpid-dispatch to /main/qpid-dispatch.
+* Builds and installs qpid-dispatch using gcc.  /usr/local/sbin is the folder in which qpid-dispatch executable is installed
+* Launch qpid-dispatch router
+
+####Building and running Dockerfiles
+* To build the fedora docker file
+ * sudo docker build -t username/dispatch-fedora:latest --file=Dockerfile-fedora  .  (substitute username with your username e.g. johndoe)
```

Comment:
I don't think the username is required here.  If a username is used, it must be the username for the Docker Index subscription (not the apache or local username).  It's ok to omit the username in the image title if there's no plan to push the image to the Docker Index.